tests/integration/targets/lke_node_pool_basic/tasks/main.yaml

```diff
@@ -107,6 +109,7 @@
       assert:
         that:
           - update_pool.node_pool.count == 1
+          - update_pool.node_pool.firewall_id == 654321
```

Comment:
Since these are integration tests that run against the API, I think the firewall ID should reflect that of an actual firewall.

Fortunately is an existing `{{ firewall_id }}` variable that we use to secure test resource: https://github.com/linode/ansible_linode/blob/cb7f6061f7174584cae6c64e66ce218820f36e87/tests/integration/targets/instance_basic/tasks/main.yaml#L27

For firewall_id update, we should be able to use the firewall module at the beginning of this test to create a temporary firewall for the sake of testing.